src/js/components/Article.js

```diff
@@ -440,7 +440,12 @@ export default class Article extends Component {
           const elementClone = React.cloneElement(element, {
             ref: index
           });
-          let elementNode = elementClone;
+          let elementNode = (
+            <div>
+              {elementClone}
+              {controls}
```

Comment:
This will put the controls in each child though, which I don't think makes the animation look right.